conductor_api/src/interface.rs

```diff
@@ -163,7 +163,8 @@ impl ConductorApiBuilder {
             let func_name = Self::get_as_string("function", &params_map)?;
 
             let cap_request = {
-                let context = hc.context();
+                let context = hc.context()
+                    .expect("Reference to dropped instance in interface handler. This should not happen since interfaces should be rebuild when an instance gets removed...");
```

Comment:
```suggestion
                    .expect("Reference to dropped instance in interface handler. This should not happen since interfaces should be rebuilt when an instance gets removed...");
```